README.md

```diff
@@ -109,3 +109,17 @@ Configure Bundler to always use the correct arguments when installing:
 Using Gem to install directly without Bundler:
 
     gem install charlock_holmes -- --with-icu-dir=/usr/local/opt/icu4c
+
+### Other
+
+RedHat, CentOS, Fedora:
+
+    yum install libicu-devel
+
+Debian, Ubuntu, Mint:
+
+    apt-get install libicu-de
```

Comment:
Is this a typo, and should read like this?

```suggestion
    apt-get install libicu-dev
```

Reply:
It's a typo. Thanks for point it out.